eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -190,11 +190,8 @@ class VolumePreparer:
         all_flatpak_refs = list(set(all_flatpak_refs))
 
         # Pass the heavy lifting off to `ostree create-usb` and `flatpak create-usb`
-        # which require an updated summary file
-        # TODO: Verify that it adds GPG keys where appropriate.
```

Comment:
I think this comment should be deleted if I understand it correctly. GPG keys are not copied to the USB but they're not supposed to be. The destination computer has to already have the remote (and its keys) configured, and if the keys need to be updated that happens via `xa.gpg-keys` which is copied as part of `ostree-metadata`.

**But** I think you should delete the comment in a separate commit since it's unrelated to the summary update stuff.

Reply:
I've addressed this in the new version I force-pushed.

Please verify my new commit message is accurate.